config/packages/dev/sulu_admin.yaml

```diff
@@ -0,0 +1,3 @@
+sulu_admin:
```

Comment:
not sure if you commited this intentionally? 🙂

Reply:
Right :see_no_evil: Have to fix that properly in a separate PR, so that I don't have to keep that file lying around in my file system :grimacing: